docs/COMPATIBILITY.md

```diff
@@ -21,6 +21,7 @@ Spegel has been tested on the following Kubernetes distributions for compatibili
 | --- | --- |
 | :green_circle: | AKS |
 | :green_circle: | Minikube |
+| :green_circle: | Kind |
```

Comment:
Make this yellow, to indicate that it requires some sort of user intervention.

Reply:
Fixed.